src/gui/torrentmodel.h

```diff
@@ -33,15 +34,19 @@
 #define TORRENTMODEL_H
 
 #include <QAbstractListModel>
+#include <QColor>
```

Comment:
All these new includes are unneeded. You can forward declare these classes and move the includes into the .cpp file.